### DISCUSSION TITLE
Feature req: Pause/Disable Automation

### DISCUSSION THREAD

**Discussion by eJonny** [netzero-labs/netzero#4] [Ideas]
There  are times, it might be because of troubleshooting or known changes to energy use patterns at home, that I would want to disable some of the automations for a short period of time and then re-enable them later. Right now I delete the automation and then re-create it later. While this is pretty easy to do, it would be a time saver if there was a way to delete the automation and re-enable it later. It might be nice to have an entry in the "Past Runs" section to indicate when an automation is created, modified, enabled, or disabled (also for troubleshooting purposes).

**Comment by zigam** [comment]
Agreed, pausing and editing and automations is needed, I'll add that soon.

**Reply by himynamesned** [reply]
Can you look to add "one off" automation too please. Long trip and need to set car to charge from grid during ow TOU and want to avoid pw drain as a one off. Doing this with backup @ 100% and no grid charging automation. Followed by an undo. 

Preset automation scenarios would helpful too.

**Comment by eJonny** [comment]
I'm using and loving the new "pause" automation feature. Thank you!
